client/packages/cli/README.md

```diff
@@ -109,15 +109,77 @@ Options:
 - **-t, --timeout \<timeout\>**: Timeout in seconds for waiting for events from the chain. Default timeout is 30 seconds
 - **-x, --export**: Export extrinsic data to a file
 
+### estimate-gas-fee
 
-### estimate
+The `estimate-gas-fee` command is used to estimate the gas fee required for an execution
 
-The `estimate` command is used to estimate the max reward with a given target amount.
+Usage:
+
+```bash
+yarn cli estimate-gas-fee [options]
+```
+
+Options:
+
+- -t, **--target \<name\>**: The target on which is execution will be executed. i.e 'eth', 'roco'
+- -a, **--action \<action\>**: The execution action i.e tass, tran, swap
+- -o, **--args \<speed mode, eth estimation param or SFX JSON string\>**: The execution arguments. It's value can be a speed mode, a EVM call estimation or a side-effect JSON string
+- -s, **--sfx \<file-path\>**: The SFX file path
+- **--signer \<address\>**: The signer's address
+
+Examples:
+```
+# Estimate the gas fee for an asset transfer on ETH target
+p cli estimate-gas-fee -t eth -a tass -o 'fast'
+
+# Estimate the gas fee an EVM call on ETH target
+p cli estimate-gas-fee -t eth -a cevm -o '{"fromAddress":"0x1234567890AbCdEfFeDcBa09876eFfEDCBA54321","toAddress":"0x9876543210FeDcBaABcDEfFeDCbA98765EDCBA12","data":"0x0000","speedMode":"fast"}'
+
+# Estimate the gas fee required for executing the side effect
+# --signer is optional, if not provided, Alice address is used
+p cli estimate-gas-fee -t roco -a tass --sfx ./transfer.json --signer "5GrwvaEF5zXb26Fz9rcQpDWS57CtERHpNehXCPcNoHGKutQY"
```

Comment:
This fails with
```
❯ pnpm cli estimate-gas-fee -t roco -a tass --sfx ./transfer.json --signer "5GrwvaEF5zXb26Fz9rcQpDWS57CtERHpNehXCPcNoHGKutQY"
```suggestion
p cli estimate-gas-fee -t roco -a tass --sfx ./transfer.json --signer "5GrwvaEF5zXb26Fz9rcQpDWS57CtERHpNehXCPcNoHGKutQY"
```
⠧ Estimating...2023-08-22 09:03:52        RPC-CORE: queryInfo(extrinsic: Bytes, at?: BlockHash): RuntimeDispatchInfo:: createType(RuntimeDispatchInfo):: Struct: failed on weight: u64:: Assertion failed
✖ Error: createType(RuntimeDispatchInfo):: Struct: failed on weight: u64:: Assertion failed
```